### PR TITLE
[Documentation] Inclusion of install two-line script for Unix

### PR DESCRIPTION
**Pull request by Ashish0804**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

closes https://github.com/yt-dlp/yt-dlp/issues/83

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Release version](https://img.shields.io/github/v/release/yt-dlp/yt-dlp?color=brightgreen&label=Release)](https://github.com/yt-dlp/yt-dlp/releases/latest)
 [![License: Unlicense](https://img.shields.io/badge/License-Unlicense-blue.svg)](LICENSE)
 [![CI Status](https://github.com/yt-dlp/yt-dlp/workflows/Core%20Tests/badge.svg?branch=master)](https://github.com/yt-dlp/yt-dlp/actions)
-[![Discord](https://img.shields.io/discord/807245652072857610?color=blue&label=discord&logo=discord)](https://discord.gg/S75JaBna)  
+[![Discord](https://img.shields.io/discord/807245652072857610?color=blue&label=discord&logo=discord)](https://discord.gg/S75JaBna)
```
Comment (pukkandan) [inline README.md:6]: The spaces is intentional

File: README.md [hunk 2]
```diff
@@ -15,95 +15,112 @@ A command-line program to download videos from youtube.com and many other [video
...
+  - [Authentication with .netrc file](#authentication-with-netrc-file)
+- [OUTPUT TEMPLATE](#output-template)
+  - [Output template and Windows batch files](#output-template-and-windows-batch-files)
+  - [Output template examples](#output-template-examples)
+- [FORMAT SELECTION](#format-selection)
+  - [Filtering Formats](#filtering-formats)
+  - [Sorting Formats](#sorting-formats)
+  - [Format Selection examples](#format-selection-examples)
+- [PLUGINS](#plugins)
+- [MORE](#more)
```
Comment (pukkandan) [inline README.md:50]: unrelated and unnecessary change

File: README.md (hunk 1 above)
Comment (Ashish0804) [reply README.md:6]: this was my vscode formatting everything on save.
I'll make a new PR with just the required changes.

Comment (pukkandan) [reply README.md:6]: dont make new PR. just add a new commit reversing these changes

**Comment by pukkandan** [comment]
#155
